torchao/experimental/ops/mps/test/test_quantizer.py

```diff
@@ -0,0 +1,104 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the license found in the
+# LICENSE file in the root directory of this source tree.
+
+from typing import Optional
+import copy
+import os
+import sys
+
+import torch
+import torchao_mps_ops
+import unittest
+
+torchao_root: Optional[str] = os.getenv("TORCHAO_ROOT")
+assert torchao_root is not None, "TORCHAO_ROOT is not set"
+
+sys.path.insert(0, torchao_root)
```

Comment:
Mark a TODO here since this should not really be needed but likely needed because we are not integrated in torchao's setup.py

Reply:
The script should work without this if you run `pip install .` from ao directory.

@manuelcandales but if you do add something like this, you should be able to use the source file's location to find torchao_root

Reply:
Yeah. Lets not rely on some env variables to be set.

Reply:
done!